Task-3(java programs)/FileThread.java

```diff
@@ -0,0 +1,44 @@
+import java.io.*;
+import java.util.Scanner;
+
+class Files extends Thread{
+	public void run(){
+		try{
+			Scanner sc = new Scanner(System.in);
```

Comment:
dont simply put the entire code in run method instead create thread instance and copy files with the help of thread instance.